src/jv_aux.c

```diff
@@ -233,6 +236,7 @@ jv jv_set(jv t, jv k, jv v) {
     jv_free(v);
     t = err;
```

Comment:
This looks like a bug in master.  We need to `jv_free(t)` in this case.

Reply:
There is a `jv_free(t)` 3 lines above this one

Reply:
Ah, duh, I'm blind.